src/styles/blocks/card.scss

```diff
@@ -0,0 +1,48 @@
+.card {
+  max-width: $card-width;
+  padding: 14px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  perspective: 100px;
```

Comment:
perspective  - why do you need this prop here?

Reply:
It for the next task. I forgot to remove it.